src/util.cpp

```diff
@@ -878,27 +878,36 @@ std::string GetWarnings(const std::string& strFor)
 {
     std::string strStatusBar;
     std::string strRPC;
+    std::string strGUI;
+    const std::string uiAlertSeperator = "<hr />";
 
     LOCK(cs_warnings);
 
-    if (!CLIENT_VERSION_IS_RELEASE)
-        strStatusBar = _("This is a pre-release test build - use at your own risk - do not use for staking or merchant applications!");
+    if (!CLIENT_VERSION_IS_RELEASE) {
+        strStatusBar = "This is a pre-release test build - use at your own risk - do not use for staking or merchant applications";
+        strGUI = _("This is a pre-release test build - use at your own risk - do not use for staking or merchant applications");
```

Comment:
Can't we reuse `strStatusBar` rather than copying the message?
(same for `fLargeWorkForkFound` and `fLargeWorkInvalidChainFound` warnings)